barretenberg/cpp/src/barretenberg/commitment_schemes/shplonk/shplonk.hpp

```diff
@@ -33,7 +33,7 @@ template <typename Curve> using OutputWitness = bb::Polynomial<typename Curve::S
  *
  * @tparam Curve EC parameters
  */
-template <typename Curve> struct ProverOutput {
+template <typename Curve> struct ShplonkProverOutput {
```

Comment:
for this review, seems I would have to scan every file affected to make sure the names are adjusted like this.

Reply:
Yeah basically. I think scrolling through https://github.com/AztecProtocol/aztec-packages/pull/4318.diff isn't *too* bad. Mainly a review like this should be best-effort, if you see any fundamental issues, and trust that either I thought things through or that we can fix it up. Overall I don't think many names had issues being top-level